### PR TITLE
Update caption

### PR DESCRIPTION
**Pull request by sareh**
Resolves discussion in this PR: https://github.com/bbc/simorgh/pull/1692#issuecomment-492414908 
Part of https://github.com/bbc/simorgh/issues/1688

**Overall change:** Ensures that `psammead-caption` styles are self-contained (does not rely on a nested `psammead-paragraph` for paragraph styles).

In Storybook, the example with multiple paragraphs will be using the correct text color & have the same spacing: http://localhost:8080/?path=/story/caption--containing-multiple-paragraphs

Paragraphs should have 24px padding between them.
Screen Shot 2019-05-14 at 22 55 38

**Code changes:**

- Add spacing for paragraphs nested within caption
This is since `psammead-caption` should not be used with a nested `psammead-paragraph` component, but just with a `p` element instead. So the padding bottom and margin reset styles are required to now be within this caption component.
-  Updating the stories example to include recommended usage - not using psammead-paragraph, but simply a p element

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-caption/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.5 | [PR#x](https://github.com/bbc/psammead/pull/x) Add spacing for nested paragraph |
```
Comment (DenisHdz) [inline packages/components/psammead-caption/CHANGELOG.md]:
```suggestion
| 1.1.5 | [PR#524](https://github.com/bbc/psammead/pull/524) Add spacing for nested paragraph |
```

**Review by DenisHdz — COMMENTED** [review]
`CHANGELOG` needs updating, other than that LGTM 👍

**Review by twinlensreflex — DISMISSED** [review]

File: packages/components/psammead-caption/CHANGELOG.md (hunk 1 above)
Comment (sareh) [reply packages/components/psammead-caption/CHANGELOG.md]: Thanks, Denis

**Review by twinlensreflex — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Pull request merged**